quarkus-debezium-parent/pom.xml

```diff
@@ -24,7 +24,6 @@
         <maven.compiler.testTarget>${debezium.java.specific.target}</maven.compiler.testTarget>
         <maven.compiler.testRelease>${debezium.java.specific.target}</maven.compiler.testRelease>
 
-        <quarkus.platform.version>3.26.3</quarkus.platform.version>
```

Comment:
@Naros 
Given that Quarkus Extension still goes thorugh intensive development I'd prefer to use
`<quarkus.platform.version>${quarkus.version.extension}</quarkus.platform.version>` and remove the ability to override the version only when all connectors are properly handled in the extension.
WDYT?